docs/streams/stream-tasks/view-streams.md

```diff
@@ -22,6 +22,122 @@ View existing streams.
 
 When you view a stream in the console, you will not see a backlog of messages. This screen does not persist, which means that when it is open, you will see new messages as they are sent. If you close the screen, any messages displayed will not be visible the next time that you open it.
```

Comment:
As this is about viewing the stream list, and not Reading msg from the stream, do we need this line?

Reply:
I need to put that info somewhere, couldn't find a better place. Got any suggestions?

Reply:
@diana-macrometa in https://docs-git-doc-723-sdk-view-streams-team-macrometa.vercel.app/docs/streams/stream-tasks/subscribe-streams
we can add a web console tab where we can add this and explain that you can check msg that are coming in stream
@dlozina what do you think

Reply:
Updated content

Reply:
I agree with @Luba89 's point. We could tell users to open a web console tab and perhaps have a screenshot of messages that are coming in.

Reply:
Added screenshot.